tutorial/basic/index.md

```diff
@@ -541,6 +541,13 @@ you need to change the *stage* using the following sbt setting:
 
 (by default, the stage is `FastOptStage`)
 
+### Compression
+
+Use gzip to compress the size of your `fullOptJS` output.
+Optionally, the sbt plugin [`sbt-web-scalajs`](https://github.com/vmunier/sbt-web-scalajs)
+can be used in combination with [`sbt-gzip`](https://github.com/sbt/sbt-gzip)
+to automatically compress Scala.js assets as part of your build pipeline.
```

Comment:
This should be phrased in a way that:
* Does not imply it is necessary as part of the tutorial steps
* For sbt-gzip, that it's only relevant once the developer actually chose to use sbt-web (or Play or something else using sbt-web).

Right now people will get confused trying to add sbt-web-scalajs to their tutorial build, which is totally not going to work given that the tutorial is a front-end-only project. There should be something like:

> When deploying Scala.js to an actual web server, you should additionally gzip the resulting .js files, as usual. The setup depends on your server stack. A common setup is through sbt-web-scalajs and sbt-gzip if you have a Play or Akka-http server, or anything stack relying on sbt-web.